projects/1.18/assets/reliquary-v1-3/reliquary/lang/zh_cn.json

```
@@ -1 +1,362 @@
+    "jei.reliquary.description.splash_potion": "有许多不同种类的飞溅药水，查看药水精华介绍获取更多信息。",
+    "jei.reliquary.description.lingering_potion": "浓缩的滞留药水，查看药水精华介绍和JEI配方获取更多药水制作信息。",
+    "jei.reliquary.description.pyromancer_staff": "烈焰术士之杖可以发射小火球（烈焰人）和火球（恶魂），用烈焰粉和火焰弹充能。\\n\\n拿着它可以完全防止恶魂或烈焰人发射的火球的伤害。此外，在它未激活时可以扑灭玩家周围的火。\\n\\n最后，你还可以将它作为打火石使用。",
+    "jei.reliquary.description.rending_gale": "按住右键，可以使玩家向着鼠标指向的方向飞行。请注意，它本身没有缓落效果。但如果你保持按住右键状态，鼠标指向地面，便可以取消摔落伤害。\\n\\n它同时还可以在玩家指向的方向推、拉实体和在雨天召唤闪电。使用Shift+滚轮切换模式。\\n\\n使用羽毛来为其充能。",
+    "jei.reliquary.description.rod_of_lyssa": "升级版钓鱼竿，可以将实体拉的更高，并且在潜行时，可以偷窃实体携带的物品。也能拉动掉在地上的物品。",
+    "jei.reliquary.description.salamander_eye": "当手持沙罗曼蛇之眼时，附近的火焰（和你自己身上的，如果适用）会被扑灭，恶魂的火球会被自动弹回，烈焰人的火球也会熄灭，但是烈焰人在你身旁时仍能伤害你。",
+    "jei.reliquary.description.serpent_staff": "向你的敌人喷出快速且粘稠的毒液，当被击中时候，生物将会被施加Debuff，同时渲染一个15秒的负面粒子效果（无害）。",
+    "jei.reliquary.description.shears_of_winter": "拥有范围剪切效果的剪刀。左键点击可以像普通剪刀一样使用。而右键使用时，则会对玩家周围进行剪切工作，该模式下，它可以剪羊毛，破坏树叶。",
+    "jei.reliquary.description.sojourner_staff": "可以存储多达1500根火把。\\n\\n可以使用它来远距离放置火把/方块，最远距离为30格，但是每距离6格，放置火把/方块的“代价”将会增加（因此，在30格外放置火把，将消耗6根火把）。\\n\\n如果同时持有虚妄提灯，提灯会尝试使用手杖中保存的火把。",
+    "jei.reliquary.description.tipped_arrow": "可以使用圣遗物的浓缩药水合成药箭，这意味着药箭的效果将比原版多得多。",
+    "jei.reliquary.description.twilight_cloak": "在低亮度的情况下，只要物品栏有斗篷存在，玩家就会隐形。隐形状态下，怪物即使被攻击也无法发现玩家。\\n\\n按住Shift+右击启用。",
+    "jei.reliquary.description.void_tear": "右击将物品栏内数量最多的物品全部存入虚空之泪中。Shift+右击可以控制其是否是自动吸收，这样你就拥有了一个可以大量存储物品的便携式存储器.你还可以用虚空之泪右键箱子（或任何容器）来存入/取出与虚空之泪内物品相同的物品",
+    "jei.reliquary.description.witch_hat": "由女巫掉落。可以用来制作万能溶剂之经，你也可以穿上它，使你看起来像个小矮妖。",
+    "jei.reliquary.description.witherless_rose": "只要不凋玫瑰在你的物品栏内，你就不会受到凋零效果。",
+    "jei.reliquary.description.wraith_node": "使用末影手杖右击幽灵节点，将其绑定至你的幽灵手杖，允许你消耗末影珍珠来传送回节点。\\n\\n注意它不能跨维度使用。",
+    "jei.reliquary.description.pedestal": "基座可以以一种花哨的方式来显示你的物品，但同时它也支持一些自动化。\\n幸运硬币-激活时，将拾取周围的掉落物和经验并存储至基座旁的容器中，未激活时，在其范围内阻止玩家和放有幸运硬币的基座拾取物品\\n英雄之章-可充当经验储罐，当激活时，会自动搜寻周围放有可修复物品的基座，对其中物品进行经验修补\\n剑-攻击范围内的生物（忽略Boss、幼年以及被驯服动物）\\n桶-挤奶，收集世界中的流体并存储至基座旁的容器中\\n剪刀-对范围内所有可收集方块和实体进行剪切工作\\n钓鱼竿-自动钓鱼，但不会拾取物品和经验，你需要使用其他东西来进行收集\\n要打开基座功能，请空手右击基座开关，或提供红石信号，或者通过在另一基座内放置红石块或红石粉来远程控制。",
+    "jei.reliquary.description.passive_pedestal": "基座（装饰性）只是一个花哨的显示你的物品的方式，没有其他功能。",
+
+    "keybind.reliquary.category": "圣遗物",
+    "keybind.reliquary.fortune_coin": "启用/禁用幸运硬币",
+    "chat.reliquary.fortune_coin.toggle": "幸运硬币切换至%s",
+    "chat.reliquary.fortune_coin.on": "开",
+    "chat.reliquary.fortune_coin.off": "关",
+    "chat.reliquary.hero_medallion.drain_levels": "右击取出%s级经验",
+    "chat.reliquary.hero_medallion.fill_levels": "玩家等级为%s级时停止吸取",
+
+    "tooltip.reliquary.absorb": "{{!colors.light_purple}}Shift+右击切换自动吸收模式。",
+    "tooltip.reliquary.drain": "{{!colors.light_purple}}Shift+右击切换排空模式。",
+    "tooltip.reliquary.place": "{{!colors.light_purple}}Shift+右击切换放置模式。",
+    "tooltip.reliquary.absorb_active": "{{!colors.purple}}自动吸收{{!colors.gray}}{{item}}。",
+
+    "enchantment.reliquary.severing": "切断",
```

```suggestion
    "enchantment.reliquary.severing": "肢解",
```
刚从匠魂那边过来，我倾向于把这个翻译成肢解。因为圣遗物的怪物掉落物或多或少都是这个怪物本身的一部分肢体或身体，所以这样更准确。更何况severing本身也有这层含义在。

---

考虑过肢解这个翻译，但感觉和匠魂重合了就没采用

---

实际上是等价的，不过恐怕很多人会误解为打错字。